src/Pelagos/Bundle/AppBundle/Command/DoiUpdateOrphanDataciteCommand.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace Pelagos\Bundle\AppBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+use GuzzleHttp\Client;
```

Comment:
It would be more apparent if you aliased this use statement `as GuzzleClient` because I was confused below what client it was.

Reply:
I think we don't have/use any conflicting classes with that same name. If you think it is confusing, we can add an alias to it.